subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -397,6 +401,43 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	big_info->payload_count_framing[4] &= ~BIT(7);
 	big_info->payload_count_framing[4] |= ((framing & 0x01) << 7);
 
+	if (encryption) {
+		const uint8_t BIG1[16] = "1GIB";
+		const uint8_t BIG2[4] = "2GIB";
+		const uint8_t BIG3[4] = "3GIB";
+		struct ccm *ccm_tx;
+		uint8_t igltk[16];
+		uint8_t gltk[16];
+		uint8_t gsk[16];
+
+		/* Fill GIV and GSKD */
+		(void)lll_csrand_get(lll_adv_iso->giv,
+				     sizeof(lll_adv_iso->giv));
+		(void)memcpy(big_info->giv, lll_adv_iso->giv,
+			     sizeof(big_info->giv));
+		(void)lll_csrand_get(big_info->gskd, sizeof(big_info->gskd));
+
+		/* Calculate GSK */
+		err = bt_crypto_h7(BIG1, bcode, igltk);
+		LL_ASSERT(!err);
+		err = bt_crypto_h6(igltk, BIG2, gltk);
+		LL_ASSERT(!err);
+		err = bt_crypto_h8(gltk, big_info->gskd, BIG3, gsk);
+		LL_ASSERT(!err);
+
+		/* Prepare the CCM parameters */
+		ccm_tx = &lll_adv_iso->ccm_tx;
+		ccm_tx->direction = 1U;
```

Comment:
Is `direction` like RX/TX or? Should it have values that are `#define`d?

Reply:
Alright, so `1U` is sent from central, and `0U` is sent from peripheral? 

IMO that sounds like predefined values that should be put into properly defined values to make it more clear to the reader

Reply:
Hope it is ok to make changes outside this PR, as the changes would impact more that the Broadcast ISO related changes.

![image](https://user-images.githubusercontent.com/6350656/214570315-501e3c5b-7206-4528-b607-5e0bfdf224b3.png)